conga-aem-definitions/src/main/templates/aem-dispatcher/author/vhost_author.partials.hbs

```diff
@@ -6,12 +6,6 @@
 {{/block}}
 
 
-{{~#block "documentRoot"}}
```

Comment:
DocumentRoot is not required, because disp_apache2 is mapped via location directive to / and handling all the requests.